build.gradle.kts

```diff
@@ -75,7 +75,10 @@ tasks.register("unitTestAll") {
         ":dd-sdk-android-timber:testReleaseUnitTest",
         ":tools:detekt:test",
         ":tools:unit:testDebugUnitTest",
-        ":tools:unit:testReleaseUnitTest"
+        ":tools:unit:testReleaseUnitTest",
```

Comment:
I am a bit worried about this task getting too big. We're going to wait a lot for our build on Gitlab, in the same time I really don't know how exactly to avoid this and also to verify that code.

Reply:
The thing is the more time passes the longer each task will be. I want to make sure that we don't break the api compatibility without knowing it beforehand.
But what we can do is run a simpler task that maps the API surface and checks for changes, much like the plugin that checks the OSS dependencies of the project.

Reply:
I created task RUMM-155 to deal with this and removed the samples from building in the CI